packages/client/hmi-client/src/components/Navbar.vue

```diff
@@ -3,15 +3,15 @@
 		<section class="header-left">
 			<router-link :to="RoutePath.Home">
 				<img
-					v-if="currentProjectId"
+					v-if="currentProjectId || isDataExplorer"
 					src="@assets/svg/terarium-icon.svg"
 					height="36"
 					alt="Terarium icon"
 				/>
 				<img v-else src="@assets/svg/terarium-logo.svg" height="36" alt="Terarium logo" />
 			</router-link>
-			<h1 v-if="currentProjectId" @click="showNavigationMenu">
-				{{ currentProjectName }}
+			<h1 v-if="currentProjectId || isDataExplorer" @click="showNavigationMenu">
+				{{ currentProjectName ?? 'Explorer' }}
```

Comment:
Use `explorerItem.label` instead of 'Explorer' string?

Reply:
This item label is for the menu not for the navigation.